statsmodels/version.py

```diff
@@ -0,0 +1,5 @@
+short_version = '0.7.0'
```

Comment:
Yup, but there are two alternatives:

 - Read `version.py` in `setup.py`.

  Requires readability of package data, something I disagree conceptually: you should not be required to touch your stuff during installation. Weird paths, weird environments, it might fail in weird situations you have no control over.
 - Write `version.py` in `setup.py`.

  Same as above but even worse: Each installation procedure changes the stuff it is about to install. So, potentially (again, in some weird edge cases), if something during install fails, you might end up with an installed package, but without `version.py`. No unit test can test for this.
- Keep them duplicate.

  Content and installation is separate. No executable code. Any redesign of the installation procedure (`setup.ini` instead of `setup.py` or whatever the future might bring) is a matter of an easy translation. Unit tests can test if `version.py` and `setup.py` agree about the version number.